events.html

```diff
@@ -62,24 +66,11 @@
             <h1>Previous Events</h1>
             <div class="event-list">
                 <div class="event">
-                    <h3>Advanced Topics of Java: Minecraft Modding Workshop @ <b>IB 1010</b></h3>
-                    <h5>6:00pm on Wednesday, February 1st, 2017</h5>
-                    Would you like a stronger understanding of Java? Do you enjoy playing games? Well now you can have both! Through this seminar you will learn how to use your Java skill to make a mod for Minecraft.
-                </div>
-                <div class="event">
                     <h3>Bowling night @ <b>Bowler Time</b></h3>
                     <h5>6:00pm on Friday, December 2nd, 2016</h5>
                     Put your Mountain Dew back in your cupboard and roll up your bag of Doritos, it's time to prove your skillz IRL in the most xtreme way possible: Bowling.
                 </div>
                 <div class="event">
-                    <h3>OSS License Workshop @ <b>IB 1010</b></h3>
```

Comment:
Ah, yeah, but it'll have less details than it does now, since it doesn't/can't link to the slides